test/dlc_tests/conftest.py

```diff
@@ -856,7 +856,7 @@ def skip_inductor_test(request):
 
 
 @pytest.fixture(autouse=True)
-def skip_dgl_test(request):
+def skip_pt20_cuda121_tests(request):
```

Comment:
Is there a reason to make this an autouse fixture? We seem to be autousing this everywhere, and then checking if the fixture is a part of the pytest markers on the test function. We might as well remove the `autouse=True`, and remove the check in line 868.

Secondly, the skip message in line 873 still says "DGL".

Reply:
addresses comment, it may not harm to have pytest.fixture(autouse=True)